src/scss/style.scss

```diff
@@ -5,3 +5,5 @@
 @import 'font-awesome-config';
 
 @import 'base';
+@import 'dashboard';
```

Comment:
The `_base.scss` file is the base for all custom CSS. This file is used for variables, vendor imports (i.e. bootstrap and font-awesome) and then importing in our custom css. Let's move this import to `_base.scss`.